_source/_posts/2022-08-30-introducing-the-new-okta-mobile-sdks.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: blog_post
+title: "Introducing the New Okta Mobile SDKs"
+author: mike-nachbaur
+by: internal-contributor
+communities: [mobile]
+description: "Announcing Okta's new, modular, streamlined mobile SDKs! Add auth in a couple lines💥"
```

Comment:
@havi, any ideas on how to make this sound better?

Reply:
"Announcing Okta's new Mobile SDKs: easier to use, way more features, and a fresh new lemon scent! 🍋"

Reply:
```suggestion
description: "Announcing Okta's new Mobile SDKs: easier to use, way more features, and a fresh new lemon scent! 🍋"
```